### DISCUSSION TITLE
[v0.4.XX] Login Page Design is too "Square-ish"

### DISCUSSION THREAD

**Discussion by goulvenb** [open-webui/open-webui#7282] [General]
Hey 👋
I've talked about it shortly on a [Discord Thread](https://discord.com/channels/1170866489302188073/1309873685791768616) but i'm writing here for the good practice.

I'm using Firefox and when using the default password manager with webapps, the background color of the `<input>` fields become yellow.
This mean that if an input fields does not have any `border-radius` nor `padding`, the render is, well, horrible.
There wasn't any problem with the older design (≤ v0.3..35) but with the new one this problem arize.
Could it be possible to patch that ?

Old render (v0.3) :
![webui_v0-3](https://github.com/user-attachments/assets/0922fc80-7ff3-4d05-85ce-dafaa64dda59)

New render (v0.4) :
![webui_v0-4](https://github.com/user-attachments/assets/2b3ee50e-58ce-44e1-8a2e-4f707f9be109)

**Comment by cstoicescu** [comment]
I have noticed it too, looks like the CSS is not applied correctly? Or is it by design?

**Reply by goulvenb** [reply]
It's by design.
I've made a [PR](https://github.com/open-webui/open-webui/pull/7284/files) which was rejected.
After asking why on the Discord, it was because the maintainer [don't want PR associated with CSS](https://discord.com/channels/1170866489302188073/1210732510250541067/1309947065098567811).

However, even though it's by design, i don't believe it's intentional and more like he made the style change without verifying how Chromium & Firefox Password Managers would react.
